pkg/controller/servicebindingrequest/planner.go

```diff
@@ -30,6 +30,7 @@ type Plan struct {
 	Name           string                      // plan name, same than ServiceBindingRequest
 	CRDDescription *olmv1alpha1.CRDDescription // custom resource definition description
 	CR             *unstructured.Unstructured  // custom resource object
+	SBR v1alpha1.ServiceBindingRequest	// service binding request
```

Comment:
Let's not use (possibly vague :) ) abbreviations in the struct, call it `ServiceBindingRequest`.

Reply:
@sbose78 we have `sbr` as a abbreviation for it in many places, btw :-) So, I would not find strange to find SBR along the code.

Reply:
> we have sbr as a abbreviation

Agreed :) I'm fine with having the usage in function scoped variables. The `Plan` struct is a very well exposed structure - and wish to have cleaner naming.